DocSum/tests/test_compose_vllm_on_rocm.sh

```diff
@@ -0,0 +1,264 @@
+#!/bin/bash
+# Copyright (C) 2024 Advanced Micro Devices, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+set -xe
+IMAGE_REPO=${IMAGE_REPO:-"opea"}
+IMAGE_TAG=${IMAGE_TAG:-"latest"}
+echo "REGISTRY=IMAGE_REPO=${IMAGE_REPO}"
+echo "TAG=IMAGE_TAG=${IMAGE_TAG}"
+export REGISTRY=${IMAGE_REPO}
+export TAG=${IMAGE_TAG}
+export MODEL_CACHE="./data"
+
+
+WORKPATH=$(dirname "$PWD")
+LOG_PATH="$WORKPATH/tests"
+ip_address=$(hostname -I | awk '{print $1}')
+
+export host_ip=${ip_address}
+export HOST_IP=${ip_address}
+export EXTERNAL_HOST_IP=${ip_address}
+export DOCSUM_HUGGINGFACEHUB_API_TOKEN="${HUGGINGFACEHUB_API_TOKEN}"
+export DOCSUM_MAX_INPUT_TOKENS=2048
+export DOCSUM_MAX_TOTAL_TOKENS=4096
+export DOCSUM_LLM_MODEL_ID="Intel/neural-chat-7b-v3-3"
+export DOCSUM_VLLM_SERVICE_PORT="8008"
+export DOCSUM_LLM_ENDPOINT="http://${HOST_IP}:${DOCSUM_VLLM_SERVICE_PORT}"
+export DOCSUM_WHISPER_PORT="7066"
+export ASR_SERVICE_HOST_IP="${HOST_IP}"
+export DOCSUM_LLM_SERVER_PORT="9000"
+export DOCSUM_BACKEND_SERVER_PORT="18072"
+export DOCSUM_FRONTEND_PORT="18073"
+export BACKEND_SERVICE_ENDPOINT="http://${EXTERNAL_HOST_IP}:${DOCSUM_BACKEND_SERVER_PORT}/v1/docsum"
+
+function build_docker_images() {
```

Comment:
Please refer to tgi test scripts, the change is due to base image update. 
![image](https://github.com/user-attachments/assets/0ac94cd3-7f9f-49a8-9388-c746ca432562)